webrtc-stats.html

```diff
@@ -2049,6 +2050,21 @@ <h2>
                   MediaStreamTrack, if <code>type</code> is <code>"track"</code>).
                 </p>
               </dd>
+              <dt>
+                <dfn><code>hugeFramesSent</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of huge frames sent by this RTCRtpSender (or for this
+                  MediaStreamTrack, if <code>type</code> is <code>"track"</code>). Huge frames, by
+                  definition, are frames with encoded size at least 2.5 times the target bitrate
```

Comment:
Suggest writing this in two parts: "encoded size at least 2.5 times the typical frame size, where the typical frame size is defined as the targetBitrate of the corresponding RTCOutboundRTPStreamStats divided by the targetBitrate of this RTCVideoSenderStats".

Reply:
I agree, this is more readable, but I still don't want to tie this to other reported stats, because there's some aggregation going there. I want to keep "the target bitrate divided by the target fps of the encoder at the time the frame was encoded." Is that fine with you?

Reply:
No. When you don't tie it to other stats in an understandable way, two implementations will implement it differently, and give different counters for the same bitstream.
If you add "at the time the frame was encoded", that's fine. It's going to give you the same numbers in two implementations (within the variability of encoding times).